docs/frontend-system/utility-apis/03-consuming.md

```diff
@@ -0,0 +1,67 @@
+---
+id: consuming
+title: Consuming Utility APIs
+sidebar_label: Consuming APIs
+# prettier-ignore
+description: Consuming utility APIs
+---
+
+All of the utility API extensions that were passed into your app through installed plugins, get instantiated and configured in the right order by the framework, and are then made available for consumption. You can interact with these instances in the following ways.
+
+## Via React hooks
+
+The most common consumption pattern for utility APIs is to call the `useApi` hook inside React components to get an implementation via its API ref. This applies whether it was originally provided from the core framework or from a plugin.
+
+```tsx
+import { useApi, configApiRef } from '@backstage/frontend-plugin-api';
+
+const MyComponent = () => {
+  const configApi = useApi(configApiRef);
+  const baseUrl = configApi.getString('backend.baseUrl');
```

Comment:
I think we should avoid this particular example since you'd want to use the discovery API for this instead. Perhaps just `app.title` or smth instead?